docs/user-guide/as923.md

```diff
@@ -0,0 +1,32 @@
+# Region AS923
+
+If you configure a concentrator with an AS923 (sub-)region, you need to follow these steps in addition to how you would typically configure a concentrator for such a region. Since in AS923, dwell time limitations may apply, you need to be aware what the local regulations are with respect to dwell time limitations. Per default, the Network Server will assume that dwell time restrictions apply. For every concentrator in that region, you must configure the device twin to include the actual dwell time limitations that apply in the location of the concentrator. You can do this by adding the following block to the desired properties of the concentrator twin:
+
+```json
+{
+    ...,
+    "properties": {
+        "desired": {
+            "routerConfig": { ... },
+            "desiredTxParams": {
+                "downlinkDwellLimit": true,
+                "uplinkDwellLimit": true,
+                "eirp": 5
+            }
+        }
+    }
+}
+```
+
+ The configuration values will be translated into a `TxParamSetupReq` MAC command (for an explanation refer to the LoRa specification) and as such be communicated automatically to every device that connects to the concentrator, based on whether that device already responded to the `TxParamSetupReq` MAC command with a `TxParamSetupAns`. In case a device did not acknowledge such a MAC command, or if the last acknowledged `TxParamSetupReq` does not match the desired TX params, it will automatically send downlink MAC commands to the device until it receives a `TxParamSetupAns`.
+
+!!! warning
+    If a device is associated with multiple concentrators, it's important that all of these concentrators have the same `desiredTxParams` properties. If they do not have the same `desiredTxParams` this will lead to unpredictable and unexpected behavior. Consider using [Automatic IoT device and module management](https://docs.microsoft.com/en-gb/azure/iot-hub/iot-hub-automatic-device-management) to manage consistent configuration between multiple concentrators.
+
+The Network Server will apply the following effective dwell time settings in this order:
+
+1. If a device acknowledged a `TxParamSetupReq`, the Network Server will always apply dwell time settings based on the last reported properties of the device. If the last acknowledged dwell time settings (reported properties) do not match what is configured on the LoRa device, make sure to update the reported properties of the device to match the actual TX params.
+2. If a device has not yet acknowledged (or received) a `TxParamSetupReq`, the Network Server will apply the regional dwell time defaults for downlink transmissions until the device acknowledges a received `TxParamSetupReq` MAC command.
```

Comment:
What do the "the regional dwell time defaults" mean exactly - no dwell time limit?

Reply:
Clarified this in ce8d735456d567024d50c8bc97b520f6ae529855. Per default dwell time limits apply if nothing else is configured in the reported properties of the device.